deploy/crds/crd-certificates.yaml

```diff
@@ -80,6 +80,12 @@ spec:
                         enum:
                           - DER
                           - CombinedPEM
+                cleanupPolicy:
```

Comment:
I just noticed that the flag talks about a "strategy" but the field talks about a "policy".

Shouldn't the flag say `--default-secret-cleanup-policy` instead of `--default-secret-cleanup-strategy`?

Alternatively, we could change the field to say `cleanupStrategy`.

I am OK with both ways, I just wish to have the flag name and the field name to be coherent.

What do you think?

Reply:
Yes, it is a good point ! I'll change --default-secret-cleanup-strategy to --default-secret-cleanup-policy  !